docs/source-pytorch/debug/debugging_basic.rst

```diff
@@ -128,6 +128,15 @@ To add the child modules to the summary add a :class:`~pytorch_lightning.callbac
 
     trainer = Trainer(callbacks=[ModelSummary(max_depth=-1)])
 
+To print the model summary if ``.fit()`` is not called:
+
+.. testcode::
```

Comment:
This won't work, because model is not defined. Let's just use the regular code-block

```suggestion
.. code-block:: python
```